src/client/dfs/duns.c

```diff
@@ -163,20 +163,23 @@ duns_resolve_lustre_path(const char *path, struct duns_attr_t *attr)
 	}
 	fd = open(path, O_RDONLY | O_DIRECTORY | O_NOFOLLOW);
 	if (fd == -1 && errno != ENOTDIR) {
-		int err = errno;
-
 		D_ERROR("unable to open '%s' errno %d(%s).\n", path, errno,
 			strerror(errno));
-		return err;
+		return errno;
```

Comment:
I'm not sure this change is progress, lots of things can change errno so we shouldn't assume it's the same after D_ERROR.  The D_ERROR itself however should be using err rather than errno.

Reply:
Right !!